lib/text/simple_text_displayer.js

```diff
@@ -82,8 +82,25 @@ shaka.text.SimpleTextDisplayer = class {
     // Flatten nested cue payloads recursively.  If a cue has nested cues,
     // their contents should be combined and replace the payload of the parent.
     const flattenPayload = (cue) => {
-      // TODO: If we want to support bold, italic, underline here, we would
-      // insert markup into the payload.
+      // Handle styles (currently underlines/italics). TODO support for color.
```

Comment:
Could you add support for bold?  Something like `cue.fontWeight >= shaka.text.Cue.fontWeight.BOLD`?